### DISCUSSION TITLE
[source-google-search-console] Problem creating new source `InvalidSiteURLValidationError`

### DISCUSSION THREAD

**Discussion by devorbit** [airbytehq/airbyte#32039] [Connector Questions] [answered]
Hi Team, 

I am currently using Airbyte OSS. I am trying to use the Google Search Console source with recommended service account setup. 

While the service account seems to have Project level Viewer and Domain wide delegation for this `https://www.googleapis.com/auth/webmasters`, it still doesn't work.

For the Admin Email field, I am using my own email where I have Viewer permissions to GSC.

Can you please help on the issue?

Error: 
```
Configuration check failed
"InvalidSiteURLValidationError('The following URLs are not permitted: quillbot.com')"
```

**Comment by aljosja-online** [comment, marked as answer]
I had the same problem and found the solution in the initial specification of the search console api. In the case of using a search console domain instead of just a specific url the website url is written like this "sc-domain:example.com" as described here https://developers.google.com/webmaster-tools/v1/sites/get. This was the solution for me.

**Comment by marcosmarxm** [comment]
Sometimes this problem can be related to the issue discussed here https://github.com/airbytehq/airbyte/issues/34285 where you're using a wrong permission and Airbyte is returning a bad error message.

[a comment by an automated account is not shown]
